aiconfig-docs/docs/overview/create-an-aiconfig.md

```diff
@@ -71,30 +71,34 @@ async function createAIConfig() {
 :::
 
 ```python title="app.py"
+import asyncio
 from aiconfig import AIConfigRuntime
 
-new_config = AIConfigRuntime.create("my_aiconfig", "This is my new AIConfig")
+async def main():
+    new_config = AIConfigRuntime.create("my_aiconfig", "This is my new AIConfig")
 
-# OpenAI completion params
-model = "gpt-4-0613"
-data = {
-    "model": model,
-    "messages": [
-      { "role": "user", "content": "Say this is a test" },
-      { "role": "assistant", "content": "This is a test." },
-      { "role": "user", "content": "What do you say?" }
-    ]
-}
+    # OpenAI completion params
+    model = "gpt-4"
```

Comment:
I'm assuming we removed the model version identifier to remove that "dependency"?